test.sh

```diff
@@ -74,6 +74,12 @@ openssl req -new -newkey rsa:4096 -days 365 -nodes -x509 \
 kubectl create namespace "$NAMESPACE"
 kubectl -n "$NAMESPACE" create secret tls certificates --cert=cert.crt --key=cert.key
 
+echo 1>&2 "Installing Prometheus Operator"
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
```

Comment:
This takes a few seconds, which matters when tests are run repeatedly. Can we only run this when necessary?

Reply:
This makes sense to me. What are the cases you think it should or shouldn't run?

Reply:
Is it even necessary? I think `helm repo add` fetches the repo contents. I checked that `helm search repo prometheus` doesn't return any results after removing the repo, and does return expected results after adding it back with `helm repo add`, but without calling `helm repo update`.

Reply:
I think `helm repo update` would only be necessary for local development where people could potentially have outdated versions of the helm repo. There might be a better way of notifying instead of forcing the update.

Reply:
If we should be concerned with updates, shouldn't we deploy a specific version of the prometheus stack?